src/apb/engine.py

```diff
@@ -117,6 +120,43 @@ def insert_encoded_spec(dockerfile, encoded_spec_lines):
     return dockerfile
 
 
+def write_playbook_file(playbooks_path, action_file, template, apb_name, action):
+    action_path = os.path.join(playbooks_path, action_file)
+    playbook_out = template.render(apb_name=apb_name, action_name=action)
+    write_file(playbook_out, action_path, True)
+
+
+def generate_playbook_files(playbooks_path, roles_path, bindable, skip, apb_name):
+    ENV = Environment(loader=FileSystemLoader(DAT_PATH))
+    template = ENV.get_template(EX_PLAYBOOK_FILE)
+
+    print("Generating playbook files")
+
+    if 'provision' not in skip:
```

Comment:
Could do a for loop here that iterates through a list of actions [provision, deprovision, bind, unbind]  since the only difference between these if statements is the $ACTION.